mesheryctl/internal/cli/root/connections/connections.go

```diff
@@ -15,7 +15,7 @@ var (
 	availableSubcommands = []*cobra.Command{listConnectionsCmd, deleteConnectionCmd}
 
 	pageNumberFlag int
-	maxRowsPerPage int
+	maxRowsPerPage = 25
```

Comment:
The pagesize should be same for all  such commands, move it into some utils or const or someexisting package and use it everywhere instaed of redefining for all commands.

Reply:
Let's merge this pull request. I'll implement this change for all commands at once.